web_handler_block.py

```diff
@@ -16,9 +16,24 @@ class WebHandler(GeneratorBlock):
                                               'microseconds': 0})
     version = VersionProperty('1.0.0')
 
+    ssl_enable = BoolProperty(title='Secure Sockets Layer (SSL) Enabled',
+                              default=False)
+    ssl_cert = StringProperty(title='SSL Certificate File',
```

Comment:
Do these have to be the file name? There is a `FileProperty` that may be more suitable for this but I think that will return a file pointer rather than just the file name.

Reply:
You're correct.  The web module needs those two file references to be strings, otherwise the service will fail to configure:

https://github.com/niolabs/nio-core/blob/b2535192b2dbe510ee97fdc34a53a8f962dad0ad/niocore/modules/web/cherrypy/server.py#L69